src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -31,24 +31,46 @@ public void promptNextPlayer(){
 
     public void playGame(){
         Scanner keyboardScanner = new Scanner(System.in);
+        while (true){
+            while (board.getWinner() == null && !(board.isTie())){
+                board.printBoard();
+                promptNextPlayer();
+                String line = keyboardScanner.nextLine();
+                String input[] = line.split(",");
+                try {
+                    board.playMove(Integer.parseInt(input[0]), Integer.parseInt(input[1]));
+                } catch (InvalidMoveException e) {
```

Comment:
Inconsistent use of whitespace surrounding brackets

Reply:
Made these consistent.